compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -641,6 +641,11 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         ErrorFollowing, EncodeCrossCrate::Yes,
         "rustc_deprecated_safe_2024 is supposed to be used in libstd only",
     ),
+    rustc_attr!(
+        rustc_pub_transparent, Normal, template!(Word),
+        WarnFollowing, EncodeCrossCrate::Yes,
+        "rustc_pub_transparent is supposed to be used in libstd only",
```

Comment:
give this a better description

(rustc_deprecated_safe_2024 also has a bad description, but that's unrelated)